player/droplayer.cpp

```diff
@@ -292,6 +292,7 @@ UINT8 DROPlayer::UnloadFile(void)
 {
 	if (_playState & PLAYSTATE_PLAY)
 		return 0xFF;
+	FileLoader_FreeData(_fLoad);
```

Comment:
The player is supposed to only release the "file handle" here.
It is not supposed to free the actual file data, as the host application is supposed to keep track of that.

So please remove that line. Same in the S98 and VGM players.